fedora/README.md

```diff
@@ -26,7 +26,7 @@ The metalink `http://mirrors.fedoraproject.org` is used to automatically select
 
 # Supported Docker versions
 
-This image is officially supported on Docker version 1.6.2.
+This image is officially supported on Docker version 1.6.2 and later.
```

Comment:
What's this Fedora that you speak of?  Is that a Docker image I can
download somewhere? :innocent: